### ISSUE TITLE
Allow to set options to presets

### ISSUE THREAD

**Issue by ktsn** [ktsn/houl#28]
To customize a preset behavior, it would be nice if we can specify preset options. The `preset` option may have an object to do that.

```js
{
  "input": "src",
  "output": "dist",
  "preset": {
    "name": "some-houl-preset",
    "options": { /* the preset options goes here */ }
  }
}
```

Then, in the `some-houl-preset`, the preset author can define `houl.config.js` with a function that receive the preset options as its 1st argument.

```js
module.exports = function config(options) {
  // Must return preset config
  return {
    rules: {
      // ...
    }
  }
}
```

**Comment by ktsn** [comment]
...and we may want to pass some options object into each task as well. Each rule object could have it.

```js
module.exports = function config(options) {
  // Must return preset config
  return {
    rules: {
      js: {
        task: 'scripts',
        options: options.js
      }
    }
  }
}
```

Task functions will receive the `options` as its 2nd argument.

```js
exports.scripts = function(stream, options) {
  return stream.pipe(buble(options))
}
```

**Comment by ktsn** [comment]
Implemented via #29
